.github/workflows/lint_python.yml

```
@@ -0,0 +1,25 @@
+name: lint_python
+on: [pull_request, push]
+jobs:
+  lint_python:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+      - run: pip install --upgrade pip wheel
+      - run: pip install bandit codespell darker flake8 flake8-2020 flake8-bugbear
+                         flake8-comprehensions isort mypy pytest pyupgrade safety
+      - run: bandit --recursive --skip B101 . || true  # B101 is assert statements
+      - run: darker --check .
+      - run: codespell || true  # --ignore-words-list="" --skip="*.css,*.js,*.lock"
```

Wow, `codespell` is cool and actually uncovered two actual spelling mistakes!

This would have to skip `.git/` and `.mypy_cache` though.

```suggestion
      - run: codespell --skip="*.json,.git" || true
```

---

The incredible thing about codespell is its speed even on a massive codebase.